apps/builddao/widget/components/Bullet.jsx

```diff
@@ -1,5 +1,6 @@
-const { theme } =
```

Comment:
Any way we can remove this coming from themeprovider/ theme?

Reply:
Well, it is an object and not a wrapper, else we could have wrapped all of our app within it